web/README.md

```diff
@@ -0,0 +1,242 @@
+# Rafiki Dashboard
+
+## How to run the app
+
+```
+yarn start
+```
+
+open https://localhost:3000/
+
+
+## How to setup Api End Point (environment parameters)
+
+```
+$ vim .env
+```
+
+```
+NODE_PATH=./src
+REACT_APP_WEBSITE_NAME="Rafiki DashBoard"
+REACT_APP_API_POINT_HOST=ncrs.d2.comp.nus.edu.sg
+REACT_APP_API_POINT_PORT=3000
```

Comment:
Should Rafiki Web read from the centralized root `.env.sh` configuration file, instead of having its own `.env`? 
Here, I'm not sure if `REACT_APP_WEBSITE_NAME` is necessary, as it is a very minor customization for the front-end.  Since `REACT_APP_API_POINT_HOST` and `REACT_APP_API_POINT_PORT` are equivalent to `.env.sh`'s `RAFIKI_ADDR` and `ADMIN_WEB_EXT_PORT`, so the front-end code can read from there instead.

Reply:
now move it to .env.sh, but the environment variable need to start with REACT_APP to avoid accidentally exposing a private key on the machine that could have the same name.

https://facebook.github.io/create-react-app/docs/adding-custom-environment-variables

Reply:
Ok i see. We can keep to your latest changes first. I'll see what I can do about this

Reply:
Hmm actually, if we consider the `web/` project independent of the rest of the Rafiki (except for its dependency on Rafiki's backend), it might be a good practice to separate the configuration files, like what you did previously...

Maybe you can focus on the front-end code; Let me handle & modify this part of the code that configures the React app.